### ISSUE TITLE
JBoss Fuse reported as "present" on machines with JBoss AMQ

### ISSUE THREAD

**Issue by kdelee** [quipucords/quipucords#1141]
## Specify type:
- Bug

### Priority:
- High
___

## Description:
The fingerprinter identifies JBoss AMQ as JBoss Fuse based on the `fuse_activemq_version`. Looks like fuse ships with some of the message queue features of AMQ but is not AMQ.

___

### Version Information:
Master branch server -- no CLI used

### Expected behavior:
JBoss AMQ is not identified as JBoss Fuse

### Actual behavior:
JBoss AMQ **is** identified as JBoss Fuse

### Steps to reproduce:
Scan test machine `sonar-jbossamq-rhel-6-vc`

### Environment information:

Scanning:

Name | Hypervisor | OS Type | OS Version |  Products installed | Products Found
---|---|---|---|---|---
sonar-jbossamq-rhel-6-vc | vCenter | Red Hat Enterprise Linux | 6.9 | JBoss AMQ 7.0.2; Open JDK 1.8| JBoss Fuse Unknown-Release: redhat-630262

```
[{'architecture': 'x86_64',
  'bios_uuid': 'C94A1E42-64A8-DFA6-E289-3AE747CEC682',
  'cpu_core_count': 1,
  'cpu_count': 1,
  'cpu_socket_count': 1,
  'entitlements': [],
  'id': 2,
  'infrastructure_type': 'virtualized',
  'ip_addresses': ['10.10.181.132'],
  'is_redhat': True,
  'mac_addresses': ['00:50:56:9E:D6:82'],
  'metadata': {'architecture': {'raw_fact_key': 'uname_processor',
                                'source_id': 3,
                                'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                                'source_type': 'network'},
               'bios_uuid': {'raw_fact_key': 'dmi_system_uuid',
                             'source_id': 3,
                             'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                             'source_type': 'network'},
               'cpu_core_count': {'raw_fact_key': 'cpu_core_count',
                                  'source_id': 3,
                                  'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                                  'source_type': 'network'},
               'cpu_count': {'raw_fact_key': 'cpu_count',
                             'source_id': 3,
                             'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                             'source_type': 'network'},
               'cpu_socket_count': {'raw_fact_key': 'cpu_socket_count',
                                    'source_id': 3,
                                    'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                                    'source_type': 'network'},
               'infrastructure_type': {'raw_fact_key': 'virt_type',
                                       'source_id': 3,
                                       'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                                       'source_type': 'network'},
               'ip_addresses': {'raw_fact_key': 'ifconfig_ip_addresses',
                                'source_id': 3,
                                'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                                'source_type': 'network'},
               'is_redhat': {'raw_fact_key': 'redhat_packages_gpg_is_redhat',
                             'source_id': 3,
                             'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                             'source_type': 'network'},
               'mac_addresses': {'raw_fact_key': 'ifconfig_mac_addresses',
                                 'source_id': 3,
                                 'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                                 'source_type': 'network'},
               'name': {'raw_fact_key': 'uname_hostname',
                        'source_id': 3,
                        'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                        'source_type': 'network'},
               'os_name': {'raw_fact_key': 'etc_release_name',
                           'source_id': 3,
                           'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                           'source_type': 'network'},
               'os_release': {'raw_fact_key': 'etc_release_release',
                              'source_id': 3,
                              'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                              'source_type': 'network'},
               'os_version': {'raw_fact_key': 'etc_release_version',
                              'source_id': 3,
                              'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                              'source_type': 'network'},
               'redhat_package_count': {'raw_fact_key': 'redhat_packages_gpg_num_rh_packages',
                                        'source_id': 3,
                                        'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                                        'source_type': 'network'},
               'subscription_manager_id': {'raw_fact_key': 'subman_virt_uuid',
                                           'source_id': 3,
                                           'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                                           'source_type': 'network'},
               'system_creation_date': {'raw_fact_key': 'date_anaconda_log',
                                        'source_id': 3,
                                        'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                                        'source_type': 'network'},
               'system_last_checkin_date': {'raw_fact_key': 'connection_timestamp',
                                            'source_id': 3,
                                            'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                                            'source_type': 'network'},
               'virtualized_type': {'raw_fact_key': 'virt_type',
                                    'source_id': 3,
                                    'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                                    'source_type': 'network'}},
  'name': 'dhcp181-3.gsslab.rdu2.redhat.com',
  'os_name': 'Red Hat Enterprise Linux Server',
  'os_release': 'Red Hat Enterprise Linux Server release 6.9 (Santiago)',
  'os_version': '6.9 (Santiago)',
  'products': [{'metadata': {'raw_fact_key': None,
                             'source_id': 3,
                             'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                             'source_type': 'network'},
                'name': 'JBoss EAP',
                'presence': 'absent',
                'version': None},
               {'metadata': {'raw_fact_key': 'fuse_activemq_version',
                             'source_id': 3,
                             'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                             'source_type': 'network'},
                'name': 'JBoss Fuse',
                'presence': 'present',
                'version': ['Unknown-Release: redhat-630262']},
               {'metadata': {'raw_fact_key': None,
                             'source_id': 3,
                             'source_name': '82a3e760-9773-4216-8473-eba0a9fde834',
                             'source_type': 'network'},
                'name': 'JBoss BRMS',
                'presence': 'absent',
                'version': None}],
  'redhat_certs': None,
  'redhat_package_count': 673,
  'report_id': 2,
  'sources': [{'id': 3,
               'name': '82a3e760-9773-4216-8473-eba0a9fde834',
               'source_type': 'network'}],
  'subscription_manager_id': 'C94A1E42-64A8-DFA6-E289-3AE747CEC682',
  'system_creation_date': '2017-07-25',
  'system_last_checkin_date': '2018-03-27',
  'virtualized_type': 'vmware',
  'vm_cluster': None,
  'vm_datacenter': None,
  'vm_dns_name': None,
  'vm_host': None,
  'vm_host_socket_count': None,
  'vm_state': None,
  'vm_uuid': None}]
```

Type            | Operating System | OS Version | Python Version | Virtualization
--------------- | ---------------- | ---------- | -------------- | --------------
Scanning System | Fedora             | 26       | 3.6            | Bare Metal
Target System   | RHEL             | 6.9        | 2.x          | VCenter

**Comment by abaiken** [comment]
I believe that this is a duplicate of #1139 (although your title & Environment info is much better!)

**Comment by kdelee** [comment]
OH yes you mentioned that in slack and then I lost track when I was going through my notes and filing issues because I ran into several problems with fuse yesterday. Feel free to close when you close the other.

**Comment by abaiken** [comment]
Closed by #1140
